source/tutorials/fmt-cdiff.rst

```diff
@@ -0,0 +1,478 @@
+
+.. question::
+    When grouping samples by "host_subject_id" and viewing the alpha rarefaction plot for the "observed_otus" metric, which subjects (if any) appear to exhibit sufficient diversity coverage (i.e., their rarefaction curves level off)? How many sequence variants appear to be present in those host subject ids?
+
+.. _`fmt cdiff taxonomy`:
+
+Taxonomic analysis
+------------------
+
+In the next sections we'll begin to explore the taxonomic composition of the samples, and again relate that to sample metadata. The first step in this process is to assign taxonomy to the sequences in our ``FeatureData[Sequence]`` QIIME 2 artifact. We'll do that using a pre-trained Naive Bayes classifier and the ``q2-feature-classifier`` plugin. This classifier was trained on the Greengenes 13_8 99% OTUs, where the sequences have been trimmed to only include 250 bases from the region of the 16S that was sequenced in this analysis (the V4 region, bound by the 515F/806R primer pair). We'll apply this classifier to our sequences, and we can generate a visualization of the resulting mapping from sequence to taxonomy.
+
+.. note:: Taxonomic classifiers perform best when they are trained based on your specific sample preparation and sequencing parameters, including the primers that were used for amplification and the length of your sequence reads. Therefore in general you should follow the instructions in :doc:`Training feature classifiers with q2-feature-classifier <../tutorials/feature-classifier>` to train your own taxonomic classifiers. We provide some common classifiers on our :doc:`data resources page <../data-resources>`, including Silva-based 16S classifiers, though in the future we may stop providing these in favor of having users train their own classifiers which will be most relevant to their sequence data.
+
+
+.. download::
+   :url: https://data.qiime2.org/2018.8/common/gg-13-8-99-515-806-nb-classifier.qza
+   :saveas: gg-13-8-99-515-806-nb-classifier.qza
+
+.. command-block::
+
+   qiime feature-classifier classify-sklearn \
+     --i-classifier gg-13-8-99-515-806-nb-classifier.qza \
+     --i-reads rep-seqs.qza \
+     --o-classification taxonomy.qza
+
+   qiime metadata tabulate \
+     --m-input-file taxonomy.qza \
+     --o-visualization taxonomy.qzv
+
+.. question::
+    Recall that our ``rep-seqs.qzv`` visualization allows you to easily BLAST the sequence associated with each feature against the NCBI nt database. Using that visualization and the ``taxonomy.qzv`` visualization created here, compare the taxonomic assignments with the taxonomy of the best BLAST hit for a few features. How similar are the assignments? If they're dissimilar, at what *taxonomic level* do they begin to differ (e.g., species, genus, family, ...)?
```

Comment:
You should note the issues with relying on NCBI BLAST taxonomy... and explain why the incomplete taxonomic assignments provided by the classifier are more reliable for short sequence reads

Reply:
Maybe just remove this question. It has caused some confusion with folks on the forum who think that that visualization takes the place of taxonomy classification.

Reply:
done